ts/Extensions/ExportData/ExportData.ts

```diff
@@ -562,26 +544,41 @@ function chartGetDataRows(
                 }
 
                 if (!rows[key]) {
-                    // Generate the row
                     rows[key] = [];
-                    // Contain the X values from one or more X axes
                     rows[key].xValues = [];
+
+                    const arr = [];
+                    for (let i = 0; i < series.chart.series.length; i++) {
+                        arr[i] = 0;
+                    }
+
+                    rows[key].pointers = arr;
+                    rows[key].pointers[series.index] = 1;
+                } else {
+                    const modifiedKey = `${key},${rows[key].pointers[series.index]}`,
+                        originalKey = key;
+
+                    if (rows[key].pointers[series.index]) {
```

Comment:
It took me a minute to get it. It would be nice to have a comment explaining we check here if it's a duplicate.